src/DIRAC/FrameworkSystem/Service/BundleDeliveryHandler.py

```diff
@@ -83,11 +83,15 @@ def updateBundles(self):
 class BundleDeliveryHandlerMixin:
     @classmethod
     def initializeHandler(cls, serviceInfoDict):
+
+        ## FIXME: move to an agent
```

Comment:
Is it OK to use `gThreadScheduler` in a tornado service?

Reply:
no

Reply:
But the gThreadSchedulerCall cannot stay in the `Mixin` class.
You can look at how the CS was done.

Reply:
So, that's why I wrote this "FIXME" note.

Reply:
Well, it should be "FIXME BEFORE I AM MERGED" no ? :laughing:

Reply:
Ouf... :)

Well the reason I didn't do it is because then I was pondering if this BundleDelivery stuff is needed at all (starting from the CAUpdateAgent).